examples/rpt/count_calculation/positions1.particle

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
It needs to have a space to get the first line of position. I thought it was because of "getline" but after a quick search it shouldn't, I will look at it.

Reply:
Done :)